MediaBrowser.Controller/Security/IAuthenticationManager.cs

```diff
@@ -0,0 +1,34 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace MediaBrowser.Controller.Security
+{
+    /// <summary>
+    /// Handles the retrieval and storage of API keys.
+    /// </summary>
+    public interface IAuthenticationManager
```

Comment:
Isn't the name a little misleading since it only handles API keys?

Reply:
My original idea was to move some of the auth logic from the user manager to here, but I can change it for now if you want

Reply:
No, if there's a plan for it that's fine